core/src/main/kotlin/com/malinskiy/marathon/execution/progress/ProgressReporter.kt

```diff
@@ -51,6 +53,10 @@ class ProgressReporter {
         }
     }
 
+    fun init(configuration: Configuration){
```

Comment:
`init` actually is a bad tone. This function looks like a setter for configuration.

Reply:
Removed.